extensions/blocks/membership-button/edit.jsx

```diff
@@ -154,6 +154,20 @@ class MembershipsButtonEdit extends Component {
 		);
 	};
 
+	renderAmount = product => {
+		const amount = formatCurrency( parseFloat( product.price ), product.currency );
+		if ( product.interval === '1 month' ) {
+			return sprintf( __( '%s /month', 'jetpack' ), amount );
+		}
+		if ( product.interval === '1 year' ) {
+			return sprintf( __( '%s /year', 'jetpack' ), amount );
```

Comment:
Should there be a space after `/`?

```suggestion
			return sprintf( __( '%s / year', 'jetpack' ), amount );
```

![image](https://user-images.githubusercontent.com/426388/58041048-61753600-7b37-11e9-9119-a86d262f43ba.png)

It looks a bit better imo.